lib/rules/function-allowed-list/index.mjs

```diff
@@ -55,11 +54,9 @@ const rule = (primary, secondaryOptions) => {
 					return;
 				}
 
-				const unprefixedName = vendor.unprefixed(node.value);
-
-				if (matchesStringOrRegExp(unprefixedName, primary)) {
+				if (matchesStringOrRegExp(node.value, primary)) {
 					// Check if function requires fallback
-					if (!matchesStringOrRegExp(unprefixedName, exceptWithoutPropertyFallback)) {
+					if (!matchesStringOrRegExp(node.value, exceptWithoutPropertyFallback)) {
```

Comment:
[suggestion] Minor. We might want to extract a local variable to avoid repeated `node.value` expressions. E.g.,

```js
const funcName = node.value;

if (matchesStringOrRegExp(funcName, primary)) {
```